Regular Expressions/File Pattern.sublime-syntax

```diff
@@ -0,0 +1,39 @@
+%YAML 1.2
+---
+# https://www.sublimetext.com/docs/file_patterns.html
+name: Sublime File Pattern
+file_extensions:
+  - sublime-file-patterns
```

Comment:
I feel like this will add future confusion, since there isn't such a file format.

Reply:
Makes sense. I'll drop it.